cfgov/jinja2/v1/_includes/molecules/global-search.html

```diff
@@ -87,6 +97,13 @@
                 </ul>
             </div>
         </form>
-        <button class="m-global-search_tab-trigger" aria-hidden="true"></button>
+        <button class="m-global-search_tab-trigger u-visually-hidden" aria-hidden="true">
+          .
```

Comment:
If this didn't work for the main menu, am I safe to assume it doesn't work here either?

Reply:
Yes, this is being covered in https://github.com/cfpb/cfgov-refresh/pull/4129

Reply:
Update this to `Collapse` and I think this PR will be good to go.

Reply:
Fixed